src/libs/Navigation/linkingConfig/FULL_SCREEN_TO_RHP_MAPPING.ts

```diff
@@ -54,6 +54,19 @@ const FULL_SCREEN_TO_RHP_MAPPING: Partial<Record<FullScreenName, string[]>> = {
         SCREENS.WORKSPACE.ACCOUNTING.XERO_EXPORT_PREFERRED_EXPORTER_SELECT,
         SCREENS.WORKSPACE.ACCOUNTING.XERO_BILL_PAYMENT_ACCOUNT_SELECTOR,
         SCREENS.WORKSPACE.ACCOUNTING.XERO_EXPORT_BANK_ACCOUNT_SELECT,
+        SCREENS.WORKSPACE.ACCOUNTING.XERO_EXPORT,
```

Comment:
Did we miss adding this earlier? Or is Xero a typo here and it should be NETSUITE?